### PR TITLE
[BE]Task #206616 Camp attendance feature

### PR DESCRIPTION
**Pull request by GitVivekHub**
# I have ensured that following `Pull Request Checklist` is taken care of before sending this PR

* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [x] Updated test cases in test-cases-tracker
* [x] Updated new API endpoint if any in common postman collection
* [x] Updated DB changes in db-tracker if any

### CODE REVIEW

File: src/src/camp/camp.service.ts [hunk 1]
```diff
@@ -1751,4 +1754,141 @@ export class CampService {
...
+			photo_1: document_name_photo1,
+			photo_2: document_name_photo2,
+		};
+
+		const response = await this.attendancesService.createAttendance(
+			camp_attendance_body,
+			req,
+			resp,
+		);
+
```
Comment (sagarkoshti1990) [inline src/src/camp/camp.service.ts:1794]: save direct document id as 
`photo_1: document_photo1, photo_2: document_photo2,`

File: src/src/camp/camp.service.ts [hunk 2]
```diff
@@ -1751,4 +1754,141 @@ export class CampService {
...
+		console.log('camp_attendance_body-->>', camp_attendance_body);
+		let attendance_array = UPDATE_TABLE_DETAILS.edit_attendance.attendance;
+		const response = await this.attendancesService.updateAttendance(
+			id,
+			camp_attendance_body,
+			[...attendance_array, 'updated_by'],
+			req,
+			resp,
+		);
```
Comment (sagarkoshti1990) [inline src/src/camp/camp.service.ts:1854]: save direct document id as 
`photo_1: document_photo1, photo_2: document_photo2,`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: src/src/camp/camp.service.ts (hunk 1 above)
Comment (GitVivekHub) [reply src/src/camp/camp.service.ts]: resolved

File: src/src/camp/camp.service.ts (hunk 2 above)
Comment (GitVivekHub) [reply src/src/camp/camp.service.ts]: resolved

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**